script/Contracts.s.sol

```diff
@@ -60,7 +60,9 @@ import {LiquidationJob, ILiquidationJob} from '@contracts/jobs/LiquidationJob.so
 import {OracleJob, IOracleJob} from '@contracts/jobs/OracleJob.sol';
 
 // --- Interfaces ---
-import {IERC20Metadata} from '@openzeppelin/token/ERC20/extensions/IERC20Metadata.sol';
+import {IERC20Metadata} from '@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol';
+import {IUniswapV3Pool} from '@uniswap/v3-core/contracts/interfaces/IUniswapV3Pool.sol';
+import {INonfungiblePositionManager} from '@uniswap/v3-periphery/contracts/interfaces/INonfungiblePositionManager.sol';
```

Comment:
why do we need it?

Reply:
ohh it has the 'createAndInitialize' method, interesting 🤔 
tho i think straight with the factory can also be achieved

Reply:
I stopped using `NonfungiblePositionManager` in favor of `UniswapV3Factory`.